src/events.js

```diff
@@ -671,57 +671,43 @@ export const addEventCheckInCompleteForm = (isCheckedIn, checkOutFlag) => {
-                            buttons: {
-                                cancel: {
-                                    text: "Cancel",
-                                    value: "cancel",
-                                    visible: true,
-                                    className: "btn btn-danger",
-                                    closeModal: true,
-                                },
-                                confirm: {
-                                    text: "Continue with Check-In",
-                                    value: 'confirmed',
-                                    visible: true,
-                                    closeModal: true,
-                                    className: "btn btn-success",
-                                }
-                            }
-                        });
 
+                        // Create a function to show the Bootstrap modal
+const showConfirmRepeatModal = () => {
+    const participantData = {
+        name: data['399159511'] + ' ' + data['996038075'],
+        checkInDate: new Date(data['331584571'][visit.concept]['840048338']).toLocaleString(),
+        collectionID: collection[0]['820476880'],
+    };
+    
+    $('#participantInfo').text('Participant ' + participantData.name);
+    $('#checkInDate').text(participantData.checkInDate);
+    $('#collectionID').text(participantData.collectionID);
+    $('#confirmRepeatModal').modal('show'); 
+    $('#continueCheckIn').on('click', function() {
+    $('#confirmRepeatModal').modal('hide'); 
+    });
+};
+
+    const confirmRepeat = showConfirmRepeatModal();
                         if (confirmRepeat === "cancel") return;
                     }
                 }
             }
 
             await checkInParticipant(data, visitConcept);
 
-            const confirmVal = await swal({
-                title: "Success",
-                icon: "success",
-                text: "Participant is checked in.",
-                buttons: {
-                    cancel: {
-                        text: "Close",
-                        value: "cancel",
-                        visible: true,
-                        className: "btn btn-default",
-                        closeModal: true,
-                    },
-                    confirm: {
-                        text: "Continue to Specimen Link",
-                        value: 'confirmed',
-                        visible: true,
-                        className: "",
-                        closeModal: true,
-                        className: "btn btn-success",
-                    }
-                },
-            });
+const showConfirmValModal = () => {
+    $('#confirmValModal').modal('show');
+
+    $('#continueSpecimenLink').on('click', function() {
```

Comment:
The continue to specimen link is no longer working because all this logic is doing is just closign the dialog.  The logic for confirmVal === confirmed should be moved to here